test/runtests.jl

```diff
@@ -553,4 +552,70 @@ end
         end
     end
 
+
+    @testset "extended filename parser" begin
+        filename = tempname()
+        a = [1 3; 2 4]
+        b = similar(a)
+        try
```

Comment:
Why don't you use `mktemp() do filename, _` instead of the `try`/`finally` block??

Reply:
This is to test the `fits_create_diskfile` function which requires the file to not exist, so I can't create it with `mktemp`.